### PR TITLE
Update build_dependencies.sh

### PR DESCRIPTION
**Pull request by sfc-gh-kwagner**

### CODE REVIEW

**Comment by sfc-gh-kwagner** [comment]
@sfc-gh-stakeda @sfc-gh-vreddy this updates the dependency build scripts to work for a fresh repo checkout and work with Centos7

File: scripts/build_dependencies.sh [hunk 1]
```diff
@@ -13,10 +13,12 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 target=Debug
 source $DIR/_init.sh $@
 source $DIR/build_openssl.sh -t $target
-source $DIR/build_curl.sh    -t $target
-source $DIR/build_awssdk.sh  -t $target
-source $DIR/build_cmocka.sh  -t Debug
+source $DIR/build_oob.sh     -t $target
```
Comment (sfc-gh-stakeda) [inline scripts/build_dependencies.sh:16]: ah ok, this was missing.

Comment (sfc-gh-stakeda) [reply scripts/build_dependencies.sh:16]: ah, you know what, this is no longer used in the CI.
```
ci/build/build.sh
```

Comment (sfc-gh-kwagner) [reply scripts/build_dependencies.sh:16]: Ah, I see. Does that script also work for local development? Also the README should be updated

**Comment by sfc-gh-stakeda** [comment]
@sfc-gh-kwagner , so basically we started building `libsnowflakeclient` in the container centos6, though, if you want to build it on centos7, that should mostly work too...

File: scripts/build_dependencies.sh (hunk 1 above)
Comment (sfc-gh-stakeda) [reply scripts/build_dependencies.sh:16]: Yeah, this is to start the build.
```
./ci/build_linux.sh
```
set `NEXUS_PASSWORD` to the ldap password.

**Comment by sfc-gh-kwagner** [comment]
> @sfc-gh-kwagner , so basically we started building `libsnowflakeclient` in the container centos6, though, if you want to build it on centos7, that should mostly work too...

I see. @sfc-gh-mmuthuraman can you try using the `ci/build/build.sh` script file to build the dependencies and let us know if that works?

This PR originally came from helping @sfc-gh-mmuthuraman build the libsnowflakeclient library since he will be doing a port of XP to ARM to test performance on AWS ARM servers

**Comment by sfc-gh-stakeda** [comment]
The debug build can start by this:
```
BUILD_TYPE=Debug ./ci/build_linux.sh
```

**Comment by sfc-gh-kwagner** [comment]
Okay I'll close this then

**Pull request closed without merging**